### ISSUE TITLE
blender simplify option dialog box misleading

### ISSUE THREAD

**Issue by cl3m3c7** [Mustard2/MustardSimplify#28]
in the popup dialog of blender simplify settings, I think we have two options:
either: remove the cancel button from the popup (is it possible?),
or: remove both buttons by using `invoke_popup ?` instead! (didn't like it though!)

reason: the settings are set on the fly, so changing props and then click cancel does nothing!

**Comment by Mustard2** [comment]
Not sure if you can remove the Cancel button :(
I have to check it though

**Comment by cl3m3c7** [comment]
I did ask in blender.chat, so I will wait sometime.
***if possible***, we could use `invoke_popup` + behavior of `invoke_props_dialog` ie: when you move your mouse away from the popup, it should not close until you left click or press ESCAPE key!

**Comment by Mustard2** [comment]
Yeah, it would be much better

**Comment by Mustard2** [comment]
Ah great, nice to know! I'll add it asap

**Comment by Mustard2** [comment]
Sorry for the long delay, waiting for Blender approval for the extension.

**Comment by Mustard2** [comment]
@cl3m3c7 I tried to check this, but all possible combinations of the invoke operators are not working.
Did you find a working version?

**Comment by cl3m3c7** [comment]
@Mustard2 no luck for me either, and I have noticed the same behavior in some extension (don't remember which one!).

I think it is not supported (yet?)

**Comment by Mustard2** [comment]
Yeah, I fear the same. I guess we can close this in the meanwhile, we can check if this will be supported later.

Thanks again! :)
